platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/AbstractStsRealm.java

```diff
@@ -292,7 +292,7 @@ private void logStsClientConfiguration(STSClient stsClient) {
       builder.append("key: " + entry.getKey() + "; value: " + entry.getValue() + "\n");
     }
 
-    LOGGER.debug(builder.toString());
+    LOGGER.debug("{}", builder);
```

Comment:
✏️  Would just `LOGGER.debug(builder);` work here instead? Or maybe `LOGGER.debug("builder: {}", builder);`?

Reply:
Changed to "builder: {}"